### PR TITLE
Vertex cli

### PR DESCRIPTION
**Pull request by PhilippeMoussalli**
PR that adds functionalities for compiling and running Vertex Pipelines with the CLI. 

After this PR is merged, i'd like to simplify a bit the compilation for both vertex and compile since there is a lot of repetition #517

### CODE REVIEW

File: src/fondant/cli.py [hunk 1]
```diff
@@ -335,6 +365,38 @@ def register_run(parent_parser):
...
+            """,
+        action="store",
+    )
+    vertex_parser.add_argument(
+        "--project-id",
+        help="""The project id of the GCP project used to submit the pipeline""",
+    )
+    vertex_parser.add_argument(
+        "--project-region",
+        help="The project region of the GCP project used to submit the pipeline ",
```
Comment (RobbeSneyders) [inline src/fondant/cli.py:382]: This is on a per pipeline basis right? I tested it and it works. So can we update the help message to reflect hits? Now it seems like the region is tied to the project.

**Review by RobbeSneyders — COMMENTED** [review]
Thanks @PhilippeMoussalli!

One small comment.

File: src/fondant/cli.py (hunk 1 above)
Comment (PhilippeMoussalli) [reply src/fondant/cli.py:382]: Indeed this is the region where the pipeline is run and can be different from the project region. Updated the description

**Comment by RobbeSneyders** [comment]
Thanks! You need to update this line from `project_region` to `region` to fix the test:
https://github.com/ml6team/fondant/blob/1c385056efb8e83a192b485e56785d1baff88340/tests/test_runner.py#L89

**Review by RobbeSneyders — APPROVED** [review]

**Pull request merged**